### ISSUE TITLE
Elements: Internal Tags

### ISSUE THREAD

**Issue by torrin47** [USEPA/EPA_Non-geo_Metadata_Editor#20]
_From @torrin47 on September 10, 2018 22:51_

Authenticated internal users will be provided with an additional picklist of tags specifying EPA organizational unit. Organizational units are listed here:

https://www.epa.gov/geospatial/epa-metadata-style-guide-keywords-and-epa-organization-names

_Copied from original issue: USEPA/EPA_Environmental_Dataset_Gateway#69_

**Comment by aergul** [comment]
@torrin47 
Where in the record would you like to store these internal tags? `keyword`?
Optional or mandatory?
Single or multiple selection?
Position on page?

**Comment by torrin47** [comment]
Yes, these are keywords, mandatory, single selection, and grouped with the other keywords.  There is no current authoritative source for these, the list shown in the linked styleguide above is already significantly out of date. Apparently this has been oft-discussed at the branch level, because of its impact on many other registries, so we're going to push for a broader solution rather than something only applicable to our tool.  Will keep you posted.

**Comment by aergul** [comment]
Any agency could use an Office/Branch Finder Utility (OBFU) to deobfuscate a given name. Given an office/branch name and a calendar date, it would resolve it to the original branch and its successor branch(es) (if any) using a versioned Office/Branch Registry.

Should we put this ticket on icebox or go ahead and implement with existing org chart and revise later for the broader solution when one becomes available?

**Comment by torrin47** [comment]
Sounds like a fantastic utility, wonder why the GSA doesn't provide such a service?  Let's pause it for a week or so, see if we get any traction with John Harman.
